### PR TITLE
Test hexbin unclipped

### PR DESCRIPTION
**Pull request by wkhchow**
- PR in conjunction with model factory (v1.4.0), boundaries (v.1.4.0)
- new boundaries (clipped/unclipped hexgrid for 1,5,10,25km and only unclipped for 50, and 100km grids)
- update indexes for new features in ES (clipped/unclipped hexbins for dsra, nhsl)
- update for psra now includes BC
- misc fixes and updates

### CODE REVIEW

**Review by anthonyfok — APPROVED** [review]
Awesome!  Looks good to me!

(the JSCPD, PYTHON_BLACK and PYTHON_ISORT "errors" are stylistic suggestions that we can safely ignore.)

File: python/add_data.sh [hunk 1]
```diff
@@ -500,20 +503,99 @@ wait_for_postgres() {
...
+	--pattern "${i}" >/dev/null &
+      sleep 2
+      pv -d "$(pidof gh)" || :
+    done
+
+    if [[ -f opendrr-boundaries.sql.00 ]]; then
+      cat opendrr-boundaries.sql.[0-9][0-9] > opendrr-boundaries.sql
+    fi
+
+  elif [[ "${ADD_DATA_B2_OVER_ARTIFACT,,}" =~ ^(true|1|y|yes|on)$ ]]; then
```
Comment (drotheram) [inline python/add_data.sh:530]: Remove for master merge

Comment (anthonyfok) [reply python/add_data.sh]: Agreed; besides other main and obvious reasons, the download-from-B2 mechanism had no real version control and thus a bit flaky, unlike downloading from GitHub Actions workflow run artifact where we can pinpoint the exact version we want to download.  It is time to move on. from my prevous quick-and-dirty hack.  :sweat_smile:

Resolved in commit 7d3ef2c

**Review by drotheram — APPROVED** [review]
Looks good to me!

**Pull request merged**